### DISCUSSION TITLE
Remove the "Recommendations" block

### DISCUSSION THREAD

**Discussion by Someone-G** [black7375/Firefox-UI-Fix#450] [Q&A]
Is it possible to remove the "Recommendations" block in the "Extras and Themes" section?
![Снимок экрана (169)](https://user-images.githubusercontent.com/108881628/182017420-1968bd8a-e21b-4a36-98da-f1872704c244.png)

**Comment by black7375** [comment]
I accidentally found a way.
- `extensions.getAddons.showPane` to `false`
